internal/filesystem/util.go

```diff
@@ -174,56 +100,3 @@ func Statfs(path string, buf *unix.Statfs_t) (err error) {
 		return err
 	}
 }
-
-// These variables are used solely by kernelHasMountinfoBug.
-var (
-	hasMountinfoBug        bool
-	checkMountinfoBugOnce  sync.Once
-	maxConsistentReadTimes = 3
-)
-
-// kernelHasMountinfoBug checks if the kernel bug that can lead to incomplete
-// mountinfo being read is fixed. It does so by checking the kernel version.
-//
-// The bug was fixed by the kernel commit 9f6c61f96f2d97 (since Linux 5.8).
-// Alas, there is no better way to check if the bug is fixed other than to
-// rely on the kernel version returned by uname.
-//
-// Copied from
-// https://github.com/kubernetes/mount-utils/blob/6f4aae5a6ab58574cac605cdd48bf5c0862c047f/mount_helper_unix.go#L204C1-L250
-// *    LICENSE: http://www.apache.org/licenses/LICENSE-2.0
-// *    Copyright The Kubernetes Authors.
-func kernelHasMountinfoBug() bool {
```

Comment:
There is a comment at the beginning of this file about `kernelHasMountinfoBug()`. Could you update that as well?

Reply:
Good catch, will do.